app/src/main/java/dk/scuffed/opengltest/pipeline/stages/GaussianBlurStage.kt

```diff
@@ -0,0 +1,47 @@
+package dk.scuffed.opengltest.pipeline.stages
+
+import android.content.Context
+import android.opengl.GLES20
+import dk.scuffed.opengltest.R
+import dk.scuffed.opengltest.gl.*
+import dk.scuffed.opengltest.pipeline.FramebufferInfo
+import dk.scuffed.opengltest.pipeline.GLOutputStage
+import dk.scuffed.opengltest.pipeline.Pipeline
+
+internal class GaussianBlurStage(context: Context, private val inputFramebufferInfo: FramebufferInfo, private val xDirection: Boolean, pipeline: Pipeline) : GLOutputStage(context, R.raw.vertex_shader, R.raw.gaussian_shader, pipeline) {
+
+    // https://www.rastergrid.com/blog/2010/09/efficient-gaussian-blur-with-linear-sampling/
+    private val gaussianOffsets = floatArrayOf(0.0f, 1.3846153846f, 3.2307692308f)
+    private val gaussianWeights = floatArrayOf(0.2270270270f, 0.3162162162f, 0.0702702703f)
```

Comment:
as the shader is already hardcoded 3x3, i dont see why we need to define the values here and pass them in through uniform.